docs/configuration.md

```diff
@@ -0,0 +1,91 @@
+# Configuration
+
+### allowUnused
+array of strings or regular expressions to match against a module name
+to determine if it is allowed to be unused
+
+Please create an [issue](https://github.com/charlierudolph/dependency-lint/issues)
+anytime you need to use this
+
+default:
+```yml
+allowUnused: []
```

Comment:
Yes, linking to the default values sounds like a good solution. Then this file can focus solely on providing useful documentation and examples.